org.jacoco.core/src/org/jacoco/core/internal/instr/ProbeCounter.java

```diff
@@ -30,7 +30,7 @@
 	@Override
 	public MethodProbesVisitor visitMethod(final int access, final String name,
 			final String desc, final String signature, final String[] exceptions) {
-		if (!"<clinit>".equals(name)) {
+		if (!InstrSupport.CLINIT_NAME.equals(name)) {
```

Comment:
@bjkail yup, already figure this out - https://github.com/jacoco/jacoco/pull/434#discussion_r74848553 :wink: